ntbs-service/wwwroot/source/app.ts

```diff
@@ -50,7 +50,7 @@ import BackLinkRetainingHistory from "./Components/BackLinkRetainingHistory";
 require("es6-promise").polyfill();
 require("./Polyfills/ArrayFromPolyfill");
 
-if (config.env === "production") {
+if (config.env === "production" || config.env === "staging") {
```

Comment:
The reason for re-requesting the review is because I added staging to the frontend sentry too, as this was only checking for production.

Reply:
I don't think this is necessary - I think we only have two front-end configurations.

In particular, the config is imported via the statement `import config from "./config/config-APP_TARGET";` and the `APP_TARGET` happens in the respective `webpack.dev.js` or `webpack.prod.js` files. Our staging environments will build the javascript bundle using `npm run build:prod` which uses `webpack.prod.js` - so I don't think this staging config would ever be used, and I don't think there's a need for it.

Reply:
I have reverted this change.